portfolio/src/main/webapp/script.js

```diff
@@ -47,6 +47,8 @@ function toggleVisibility (sectionToToggle) {
 
 async function fetchGreeting() {
   const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('fetched-greeting').innerHTML = greeting;
+  const greetings = await response.json();
+  console.log(greetings);
```

Comment:
Is this log needed? Generally, unless there is an explicit reason (like logging metrics), we should remove logs from code.